### PR TITLE
Ce metadata changes

### PR DESCRIPTION
**Pull request by fearthecowboy**

### CODE REVIEW

File: ce/ce/amf/contact.ts [hunk 1]
```diff
@@ -17,6 +17,8 @@ export class Contact extends Entity implements IContact {
   /** @internal */
   override *validate(): Iterable<ValidationError> {
     yield* super.validate();
+    yield* this.validateChildKeys(['email', 'role']);
+    yield* this.validateChild('email', 'string');
```
Comment (gcampbell-msft) [inline ce/ce/amf/contact.ts:21]: What is this for?

File: ce/ce/amf/metadata-file.ts [hunk 2]
```diff
@@ -52,7 +52,9 @@ export class MetadataFile extends BaseMap implements Profile {
       content = '{\n}';
     }
     const doc = parseDocument(content, { prettyErrors: false, lineCounter: lc, strict: true });
-    return new MetadataFile(doc, filename, lc, registry).init(session);
+    const result = new MetadataFile(doc, filename, lc, registry).init(session);
+    (await result).validationErrors;
```
Comment (gcampbell-msft) [inline ce/ce/amf/metadata-file.ts]: Do we still need this? Why is this here?

File: ce/ce/artifacts/activation.ts [hunk 3]
```diff
@@ -1,67 +1,561 @@
...
+        const v = findCaseInsensitiveOnWindows(this.#defines, member);
+        if (v !== undefined) {
+          return this.resolveVariables(v, locals, refcheck);
+        }
+        break;
+      }
+
+      case 'aliases': {
+        const v = findCaseInsensitiveOnWindows(this.#aliases, member);
+        if (v !== undefined) {
```
Comment (gcampbell-msft) [inline ce/ce/artifacts/activation.ts:382]: Seems like we could pull this out to the bottom or into a method, since it's duplicated in all these cases

**Review by gcampbell-msft — APPROVED** [review]

File: ce/ce/amf/metadata-file.ts (hunk 2 above)
Comment (fearthecowboy) [reply ce/ce/amf/metadata-file.ts]: removed

File: ce/ce/amf/contact.ts (hunk 1 above)
Comment (fearthecowboy) [reply ce/ce/amf/contact.ts:21]: verifying that the members have the correct type
